android cross builds are failing on CI with nightly rust

Android cross builds are failing on CI with nightly rust. This started between [build 351](https://github.com/strawlab/iana-time-zone/actions/runs/3214907058) and [build 352](https://github.com/strawlab/iana-time-zone/actions/runs/3220695414) (the macOS failures were another unrelated issue #72 which has now been solved).

The key error from the CI logs seems to be:

```
aarch64-linux-android/bin/ld: cannot find -lunwind
```

--- a/.github/workflows/rust.yml
+++ b/.github/workflows/rust.yml
@@ -147,6 +147,10 @@ jobs:
         env:
           RUSTC_BOOTSTRAP: 1
       - run: which cross || cargo +stable install cross
+      # See https://github.com/strawlab/iana-time-zone/issues/80
+      # and https://github.com/rust-lang/rust/pull/85806
+      - run: echo '[build]' > Cross.toml && echo 'build-std = true' >> Cross.toml
+        if: matrix.toolchain == 'nightly'
       - run: cross build --target ${{ matrix.target }} --examples
 
   build-ios-cross:

